version-history.md

```diff
@@ -6,6 +6,25 @@ search:
 
 This version history has been driven by what we hear from our wonderful community of users. Please share your comments or ideas in the [issue queue](https://github.com/ddev/ddev/issues). We listen. Or talk to us in any of the [support locations](https://ddev.readthedocs.io/en/stable/#support).
 
+## v1.24 (Released 2024-12)
+
+- [x] PHP 8.4, PHP 8.3 becomes default version on new projects
+- [x] NodeJS 22 becomes default
+- [x] Extensive support for matching `mysql`, `mariadb`, and `pq` clients so the client matches the server.
+- [x] MariaDB 11.4 LTS support
+- [x] Traditional Windows ARM64 (Windows Copilot) support
+- [x] `ddev_version_constraint` option added for both `config.yaml` and Add-on `install.yaml`
+- [x] `ddev get` changed to `ddev add-on get`
+- [x] Traefik router is default, and supports hosting mode, with Let's Encrypt.
+- [x] `ddevcd` command allows easy switching between project directories
+- [x] `nginx-proxy` router type removed
+- [x] `python` and `django4` project types removed
+- [x] `ddev auth ssh` can be used with individual files and against symlinks.
+- [x] 25% smaller `ddev-webserver` Docker image
+- [x] `drupal` project type introduced as an alias for latest-stable Drupal version
```

Comment:
strictly speaking, the project type `drupal` was already introduced in, was it 1.23.0(?), wouldn't it be more accurate to use something like "becomes"?

Reply:
This is taking v1.24 as a whole, all the things from v1.23.1 up to v1.24.0, so the detail doesn't matter much.